include/picongpu/fields/MaxwellSolver/Lehe/Derivative.hpp

```diff
@@ -120,13 +120,16 @@ namespace lehe
             float_64 const coeff = stepRatio * math::sin(
                 pmacc::math::Pi< float_64 >::halfValue *
                 float_64( SPEED_OF_LIGHT ) * float_64( DELTA_T ) /
-                float_64( cellSize[ T_direction ] )
+                float_64( cellSize[ dir0 ] )
```

Comment:
Note: this is unrelated, `dir0` is same as `T_direction`, just for more uniformity inside this function.